package.json

```diff
@@ -4,10 +4,13 @@
     "prestatic": "rimraf static",
     "static": "lerna run --scope @coorpacademy/components --scope @coorpacademy/app-player static -- --production",
     "poststatic": "mkdirp static && cp -r packages/@coorpacademy-components/static static/components && cp -r packages/@coorpacademy-app-player/static static/app-player",
+    "flow": "lerna run flow",
     "codecov": "codecov",
     "publish:beta": "lerna publish --npm-tag beta --exact",
     "publish:canary": "lerna publish --canary --exact --cd-version=patch --skip-git",
-    "publish:latest": "lerna publish --exact"
+    "publish:latest": "lerna publish --exact",
+    "prepare-flow": "sh scripts/links-for-nested-flow.sh",
```

Comment:
I think you should use `preflow`

Reply:
ok, but we need to keep it `postinstall` as well, kind of bootstrap, in order to be able to run `flow` within each package immediately